lib/puppet/provider/iis_site/webadministration.rb

```diff
@@ -1,18 +1,18 @@
-require 'pathname'
+require 'puppet/provider/iis_powershell'
 
 # When writing IIS PowerShell code for any of the methods below
 # NEVER EVER use Get-Website without specifying -Name. As the number
 # of sites on a server increases, Get-Website will take longer and longer
 # to return. This will exponentially increase the total duration of your
 # puppet run
 
-Puppet::Type.type(:iis_site).provide(:webadministration) do
+Puppet::Type.type(:iis_site).provide(:webadministration, parent: Puppet::Provider::IIS_PowerShell) do
   desc "IIS Provider using the PowerShell WebAdministration module"
 
   require Pathname.new(__FILE__).dirname + '../../../' + 'puppet_x/puppetlabs/iis/powershell_manager'
   require Pathname.new(__FILE__).dirname + '../../../' + 'puppet_x/puppetlabs/iis/powershell_common'
```

Comment:
these two lines shouldn't be needed now that the psmanager stuff has been moved to a parent class